### PR TITLE
Twig 2 is compatible with PHP 7.0

### PR DESCRIPTION
**Pull request by nicolas-grekas**
7.1 was needed for #3102, but this is not the case anymore since #3113

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,28 +6,34 @@ cache:
         - extra/*/vendor
         - $HOME/.composer/cache/files
 
+env:
+    global:
+        - SYMFONY_PHPUNIT_REMOVE_RETURN_TYPEHINT=1
+        - SYMFONY_PHPUNIT_DISABLE_RESULT_CACHE=1
```
Comment (nicolas-grekas) [inline .travis.yml:12]: Except for this line which can be kept, this commit should be basic reverted when merging in 3.x.

**Review by nicolas-grekas — COMMENTED** [review]
Now green :)

File: .travis.yml [hunk 2]
```diff
@@ -6,28 +6,34 @@ cache:
...
-    - (cd extra/cssinliner-extra && travis_retry composer install)
-    - (cd extra/html-extra && travis_retry composer install)
-    - (cd extra/inky-extra && travis_retry composer install)
-    - (cd extra/intl-extra && travis_retry composer install)
-    - (cd extra/markdown-extra && travis_retry composer install)
+    - ([[ $TRAVIS_PHP_VERSION = 7.0 ]] || cd extra/cssinliner-extra && travis_retry composer install)
+    - ([[ $TRAVIS_PHP_VERSION = 7.0 ]] || cd extra/html-extra && travis_retry composer install)
+    - ([[ $TRAVIS_PHP_VERSION = 7.0 ]] || cd extra/inky-extra && travis_retry composer install)
+    - ([[ $TRAVIS_PHP_VERSION = 7.0 ]] || cd extra/intl-extra && travis_retry composer install)
+    - ([[ $TRAVIS_PHP_VERSION = 7.0 ]] || cd extra/markdown-extra && travis_retry composer install)
```
Comment (stof) [inline .travis.yml:23]: it might be more readable to override the `install` and `script` sections in the 7.0 job instead.

**Comment by fabpot** [comment]
Thank you @nicolas-grekas.

**Pull request merged**